src/Autodesk.Forge.Core.E2eTestHelpers/Autodesk.Forge.Core.E2eTestHelpers.csproj

```diff
@@ -1,20 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net5.0</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <Authors>Autodesk Forge Team</Authors>
     <Company>Autodesk</Company>
     <Product>Autodesk Forge</Product>
     <Description>Shared code for Forge client sdks e2e tests</Description>
     <Copyright>Autodesk Inc.</Copyright>
-    <Version>1.0.3</Version>
+    <Version>2.0.0</Version>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/Autodesk-Forge/forge-api-dotnet-core</PackageProjectUrl>
-    <PackageIconUrl>http://developer.static.autodesk.com/images/logo_forge-2-line.png</PackageIconUrl>
+    <PackageIcon>logo_forge-2-line.png</PackageIcon>
     <PackageReleaseNotes>For full release notes see https://github.com/Autodesk-Forge/forge-api-dotnet-core/blob/master/CHANGELOG.md</PackageReleaseNotes>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Include="..\..\images\logo_forge-2-line.png" Pack="true" PackagePath="\"/>
+  </ItemGroup>
+
```

Comment:
try to get rid of warning: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048
>The 'PackageIconUrl'/'iconUrl' element is deprecated. Consider using the 'PackageIcon'/'icon' element instead. Learn more at https://aka.ms/deprecateIconUrl